js/src/util/index.js

```diff
@@ -153,6 +153,22 @@ const isVisible = element => {
   return false
 }
 
+const isDisabled = element => {
+  if (!element || element.nodeType !== Node.ELEMENT_NODE) {
+    return true
+  }
+
+  if (element.classList.contains('disabled')) {
+    return true
+  }
+
+  if (typeof element.disabled !== 'undefined') {
+    return element.disabled
+  }
+
+  return !['false'].includes(element.getAttribute('disabled'))
```

Comment:
This check is wrong AFAICT. Also should just be an equal to string check.

Reply:
it is not wrong, although could be better do a simple string check. would be the same